containers/images/core/container-assets/build/01_yum

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash -e
+
+# Setting up repositories
+echo "tsflags=nodocs" >> /etc/yum.conf
+
+# enabling SCL and EPEL
+yum -y install centos-release-scl epel-release
+
+# Installing software collections and required packages
+yum -y install rh-ruby24 rh-nodejs8-nodejs \
+    rh-ruby24-ruby{,gems} rh-ruby24-rubygem-{bundler,rdoc} \
```

Comment:
Out of curiosity, did you try the scl S2I image?

Reply:
I did a while ago, and it kept failing in different parts of the app, maybe worth revisting, my wip is at https://github.com/theforeman/foreman/compare/develop...ohadlevy:openshift

Reply:
@ehelms I just re-read what you wrote, turns out i thought you meant openshift s2i process vs the actual container. I had another look, and it might be usable, but its also not that different to our builds... thoughts?